app.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const app = express()
+const { getTopics } = require('./controllers/topics.controllers')
+
+app.use(express.json())
```

Comment:
Don't need this middleware yet for this ticket. Please remove any code that is not related to this ticket. You can always bring it back